lib/screens/config/v2/reconstructed_alert.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Screens.Config.V2.ReconstructedAlert do
+  @moduledoc false
+
+  @type t :: %__MODULE__{alert: String.t()}
```

Comment:
Do you have something in mind that this key will hold? Or is this more of a placeholder?

Reply:
Placeholder.  Right now, I only want to display _something_ about an alert for proof-of-concept, and currently that something is the alert header, a string.